Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/RenamingPolicy.cs

```diff
@@ -0,0 +1,312 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Frends.SFTP.UploadFiles.Definitions
+{
+    ///<summary>
+    /// Policies for creating names for remote files: expands macros etc.
+    ///</summary>
+    public class RenamingPolicy
```

Comment:
This can be internal class and as such most comments are not necessary as they just state the obvious